packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -82,6 +82,43 @@ export const bm25s = (model: ModelData): string[] => [
 retriever = BM25HF.load_from_hub("${model.id}")`,
 ];
 
+export const depth_anything_v2 = (): string[] => {
+	const shellCommands = [
+		`# Install the Depth-Anything-V2 library
+git clone https://github.com/DepthAnything/Depth-Anything-V2
+cd Depth-Anything-V2
+pip install -r requirements.txt`,
+	];
+
+	const pythonCode = [
+		`# Load the model and infer depth from an image
+import cv2
+import torch
+
+from depth_anything_v2.dpt import DepthAnythingV2
+
+DEVICE = 'cuda' if torch.cuda.is_available() else 'mps' if torch.backends.mps.is_available() else 'cpu'
```

Comment:
We try to be more minimal with code snippets, maybe assume cuda (few less lines)